src/components/Hero.jsx

```diff
@@ -0,0 +1,161 @@
+import { PersonalizedText } from './shared/typography';
+import { Paragraph } from './shared/typography';
+
+const Hero = () => {
+	return (
+		<>
+			<div className='flex flex-col md:flex-row'>
+				<div className='w-[100%] md:w-1/2 '>
+					<div className='h-1/3 flex flex-col justify-end ml-2' style={{ color: '#FF4409' }}>
+						<PersonalizedText>Welcome family...</PersonalizedText>
+					</div>
+					<div className='h-1/3'>
+						<Paragraph style={{ fontSize: '75px', fontWeight: '500', letterSpacing: '0.75px', lineHeight: 'normal' }}>
+							Lorem Ipsum Dolar sit Amet Luyto

```

Comment:
Half of the text is using this https://github.com/dscnitrourkela/project-udon/blob/a853ba0bdf12959d6fa6d0be3f2fb76dacdc52cf/src/components/shared/typography/index.jsx#L7